src/components/AlertMessage.jsx

```diff
@@ -29,5 +30,13 @@ class AlertMessage extends React.Component {
 		);
 	}
 }
+AlertMessage.propTypes = {
+	options: PropTypes.shape({
+		label: PropTypes.string.isRequired,
+		name: PropTypes.string.isRequired,
+		value: PropTypes.string.isRequired,
+		type: PropTypes.string.isRequired,
+	}),
```

Comment:
AlertMessage doesn't have any of this stuff, AlertMessage just has:
```
const { type, children } = this.props;
```
so you should only need 2 propTypes keys for type, and children.